cypress/component/ErrorBoundary.cy.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ErrorBoundary from '../../packages/module/dist/esm/ErrorBoundary';
+
+describe('ErrorBoundary', () => {
+  it('renders the ErrorBoundary ', () => {
+    cy.mount(<ErrorBoundary headerTitle="My app header" errorTitle="Something wrong happened"/>)
+    // cy.get('div h1').should('have.text', 'Something wrong happened');
```

Comment:
Why is the assertion commented out? Without it we probably even don't need to run the test

Reply:
For some reason ErrorBoundary is not getting rendered. I'm not sure what's going on. Could we make fixing this as part of the next ticket?

Reply:
@InsaneZein sounds good, can you please add a description of the issue to that ticket? Thank you